docs/drivers.rst

```diff
@@ -1,252 +1,32 @@
-    SHT31-D Temperature and Humidity <https://circuitpython.readthedocs.io/projects/sht31d/en/latest/>
-    Si7021 Temperature and Humidity <https://circuitpython.readthedocs.io/projects/si7021/en/latest/>
-    Thermistor Temperature <https://circuitpython.readthedocs.io/projects/thermistor/en/latest/>
-    TMP007 Contactless Temperature <https://circuitpython.readthedocs.io/projects/tmp007/en/latest/>
-    MLX90614 Contactless Temperature <https://circuitpython.readthedocs.io/projects/mlx90614/en/latest/>
-
-Light Sensors
----------------
-
-These sensors detect light related attributes such as ``color``, ``light`` (unit-less), and
-``lux`` (light in SI lux).
-
-.. toctree::
-
-    APDS9960 Proximity, Light, RGB, and Gesture <https://circuitpython.readthedocs.io/projects/apds9960/en/latest/>
-    AS726x Color Spectrum Sensor <https://circuitpython.readthedocs.io/projects/as726x/en/latest/>
-    TCS34725 Color Sensor <https://circuitpython.readthedocs.io/projects/tcs34725/en/latest/>
-    TSL2561 Light Sensor <https://circuitpython.readthedocs.io/projects/tsl2561/en/latest/>
-    TSL2591 High Dynamic Range Light Sensor <https://circuitpython.readthedocs.io/projects/tsl2591/en/latest/>
-    VCNL4010 Proximity and Light <https://circuitpython.readthedocs.io/projects/vcnl4010/en/latest/>
-    VEML6070 UV Index <https://circuitpython.readthedocs.io/projects/veml6070/en/latest/>
-    VEML6075 UV Index <https://circuitpython.readthedocs.io/projects/veml6075/en/latest/>
-
-Distance Sensors
-------------------
-
-These sensors measure the ``distance`` to another object and may also measure light level (``light`` and ``lux``).
-
-.. toctree::
-
-    Garmin LIDARLite I2C <https://circuitpython.readthedocs.io/projects/lidarlite/en/latest/>
-    TFmini IR Time of Flight ~30cm - 12m <https://circuitpython.readthedocs.io/projects/tfmini/en/latest/>
-    VL6180x 5 - 100 mm <https://circuitpython.readthedocs.io/projects/vl6180x/en/latest/>
-    VL53L0x ~30 - 1000 mm <https://circuitpython.readthedocs.io/projects/vl53l0x/en/latest/>
-    HC-SR04 Ultrasonic Range Sensors <https://circuitpython.readthedocs.io/projects/hcsr04/en/latest/>
-
-Radio
---------
-
-These chips communicate to other's over radio.
-
-.. toctree::
-
-    Adafruit Bluefruit LE SPI Friend <https://circuitpython.readthedocs.io/projects/bluefruitspi/en/latest/>
-    RFM9x LoRa <https://circuitpython.readthedocs.io/projects/rfm9x/en/latest/>
-    RFM69 Packet Radio <https://circuitpython.readthedocs.io/projects/rfm69/en/latest/>
-    PN532 NFC/RFID <https://circuitpython.readthedocs.io/projects/pn532/en/latest/>
-
-IO Expansion
---------------
+- The Adafruit bundles are available on GitHub: <https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases>.
 
-These provide functionality similar to `analogio`, `digitalio`, `pulseio`, and `touchio`.
+- Documentation for each library is available here: <https://circuitpython.readthedocs.io/projects/bundle/en/latest/>.
```

Comment:
"each library" -> "all the libraries in the bundle"

Reply:
I would take it one step further and say `documentation for the library bundle`. It's not really docs for all the libraries either.